operations/update.go

```diff
@@ -53,96 +53,99 @@ func Update() cli.Command {
 			if err != nil {
 				return errors.Wrap(err, "problem loading configuration")
 			}
+			return DoUpdate(conf, ctx, doInstall, forceUpdate)
```

Comment:
Should users have an option to set the auto-upgrade config setting when calling `evergreen get-update` rather than manually editing their YAML?

Reply:
Sure, I like the sound of that - going to add